### PR TITLE
add prepaid billing FAQs

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Prepaid billing FAQs

Screen Shot 2023-03-06 at 11 39 21

# Reasons
Prepaid Billing launch

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_includes/snippets/faq/architecture-faq-snip.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ CircleCI supports `amd64` for Docker jobs, and both `amd64` and xref:using-arm#[
 [#ipv6-in-tests]
 === Can I use IPv6 in my tests?
 
-You can use the xref:configuration-reference/#machine[Machine executor] for testing local IPv6 traffic. Unfortunately, we do not support IPv6 internet traffic, as not all of our cloud providers offer IPv6 support.
+You can use the xref:configuration-reference#machine[Machine executor] for testing local IPv6 traffic. Unfortunately, we do not support IPv6 internet traffic, as not all of our cloud providers offer IPv6 support.
```
Comment (rosieyohannan) [inline jekyll/_includes/snippets/faq/architecture-faq-snip.adoc:18]: This link was broken and causing my local build to fail, which is odd cos it's not new!

File: jekyll/_includes/snippets/faq/billing-faq-snip.adoc [hunk 2]
```diff
@@ -7,20 +7,6 @@ For example, the 25,000 credit package would provide 2,500 build minutes when us
 
 When applicable, build time can be further reduced by using parallelism, which splits the job into multiple tests that are executed at the same time. With 2x parallelism, a build that usually runs for 2,500 minutes could be executed in 1,250 minutes, further improving developer productivity. Note that when two executors are running in parallel for 1,250 minutes each, total build time remains 2,500 minutes.
 
-[#share-plans-across-organizations-billed-centrally]
```
Comment (rosieyohannan) [inline jekyll/_includes/snippets/faq/billing-faq-snip.adoc:10]: These are just moved further down

File: jekyll/_includes/snippets/faq/billing-faq-snip.adoc [hunk 3]
```diff
@@ -91,30 +162,6 @@ NOTE: If your project is xref:oss#[open-source] you will *not* be considered an
 
 To find a list of your Active Users, log in to the CircleCI web app, click menu:Plan[Plan Usage > Users].
 
-[#un-out-of-credits]
-=== What happens when I run out of credits?
```
Comment (rosieyohannan) [inline jekyll/_includes/snippets/faq/billing-faq-snip.adoc:95]: Moved up in the section for better flow

File: jekyll/_includes/snippets/faq/billing-faq-snip.adoc [hunk 4]
```diff
@@ -38,6 +24,91 @@ You can choose to pay for premium features per active user, compute, and optiona
...
+=== How do I pay?
+
+You can pay from inside the CircleCI app for monthly pricing.
+
+[#when-do-i-pay]
+=== When do I pay?
+
+On the *Performance plan*, at the beginning of your billing cycle, you will be charged for premium support tiers and your monthly credit allocation. Any subsequent credit refills _during_ the month (such as the auto-refilling at 25% on reaching 2% of credits available) will be paid _at the time of the refill_.
+
+[#what-is-prepaid-billing]
```
Comment (rosieyohannan) [inline jekyll/_includes/snippets/faq/billing-faq-snip.adoc:51]: New content from here to ......

File: jekyll/_includes/snippets/faq/billing-faq-snip.adoc [hunk 5]
```diff
@@ -38,6 +24,91 @@ You can choose to pay for premium features per active user, compute, and optiona
...
+You can follow these steps:
+
+. Select **Plans > Manage Subscription** in the web app
+. Select to **Edit** your subscription Prepaid Billing level
+
+[#do-i-still-get-bonus-credits-and-5-free-seats]
+=== Do I still get 30,000 bonus credits and 5 free seats on Prepaid Billing?
+
+Yes, all Prepaid Billing customers still receive the benefit of 30,000 credits and 5 free user seats every month.
+
```
Comment (rosieyohannan) [inline jekyll/_includes/snippets/faq/billing-faq-snip.adoc:97]: here

File: jekyll/_includes/snippets/faq/billing-faq-snip.adoc [hunk 6]
```diff
@@ -38,6 +24,91 @@ You can choose to pay for premium features per active user, compute, and optiona
...
+[#when-do-i-pay]
+=== When do I pay?
+
+On the *Performance plan*, at the beginning of your billing cycle, you will be charged for premium support tiers and your monthly credit allocation. Any subsequent credit refills _during_ the month (such as the auto-refilling at 25% on reaching 2% of credits available) will be paid _at the time of the refill_.
+
+[#what-is-prepaid-billing]
+=== What is Prepaid Billing?
+
+Prepaid Billing is a payment option for customers on Performance plan. Prepaid Billing allows you to purchase credits in bulk, pay upfront, and, based on volume purchased, be able to get discounted prices for credits.
```
Comment (kbatuigas) [inline jekyll/_includes/snippets/faq/billing-faq-snip.adoc:54]:
```suggestion
Prepaid Billing is a payment option for customers on the Performance plan. Prepaid Billing allows you to purchase credits in bulk, pay upfront, and be able to get discounted prices for credits based on volume purchased.
```
For easier reading

**Review by kbatuigas — CHANGES_REQUESTED** [review]
Just one suggestion!

**Review by kbatuigas — APPROVED** [review]

**Pull request merged**